force-app/main/default/lwc/mortgage/__tests__/mortgage.test.js

```diff
@@ -0,0 +1,40 @@
+import { getTermOptions, calculateMonthlyPayment } from 'c/mortgage';
+
+describe('c-mortgage', () => {
+    describe('calculateMonthlyPayment function', () => {
+        it('returns monthly payment with valid inputs', () => {
+            const principal = 50000;
```

Comment:
If it's a const for representing fixed test value it should be `const PRINCIPAL = ...` (same for subsequent variables).

Reply:
fixed